static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-gun-campaign.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+import config from 'lib/config';
+
+const campaignTag = '/us-news/series/break-the-cycle';
+
+const tagsMatch = () => {
+    const pageKeywords = config.page.nonKeywordTagIds;
```

Comment:
Better written as `config.get('page.nonKeywordTagIds', '').split(',')` because it's typesafe.